### DISCUSSION TITLE
test on the wild

### DISCUSSION THREAD

**Discussion by ireneb612** [Svito-zar/gesticulator#55] [Q&A] [answered]
Is it possible to do some inference in the wild? and not using only the test data?

**Comment by Svito-zar** [comment, marked as answer]
Yes, `demo` folder provides a pretrained model you can test on your own audio and text data:
https://github.com/Svito-zar/gesticulator/tree/master/demo

**Reply by ireneb612** [reply]
A stupid question, do you predict 2D or 3D coordinate?

**Reply by Svito-zar** [reply]
we predict 3D joint angle rotations

**Reply by ireneb612** [reply]
It would be nice to have a blender script that takes the rotations and moves an avatar! I know you are not an expert of Blender since I read it on an issue, but maybe some could help!

**Reply by Svito-zar** [reply]
there is actually already a script for that we developed for the GENEA Challenge 2020:
https://github.com/jonepatr/genea_visualizer

**Reply by ireneb612** [reply]
What if I would like to run blender on my own PC?

Do you think I could run the genea_visualizer/celery-queue/blender_render.py ?

**Reply by Svito-zar** [reply]
yeah, I guess so. if you experience issues with that - you can create an issue in that visualizer repo

**Reply by Svito-zar** [reply]
and if you have Blender running on your PC - you could also simply import the BVH there and play it

**Reply by ireneb612** [reply]
When using the demo.py however I only get the mp4 file, how do I get the bhv?

**Reply by Svito-zar** [reply]
@ireneb612 , it might be the easiest to fully reproduce the system, then you can get a BVH file
